src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -70,16 +70,46 @@ export default {
         default:
           break;
       }
+
+      this.$emit('error', null);
```

Comment:
drop error should be not here, in `captureMedia` method at the beginning

Reply:
done